packages/mui-base/src/BadgeUnstyled/BadgeUnstyled.tsx

```diff
@@ -1,8 +1,8 @@
 import * as React from 'react';
 import PropTypes from 'prop-types';
 import { OverridableComponent } from '@mui/types';
+import useBadge from '@mui/base/useBadge';
```

Comment:
When importing from the same package, we use relative paths (except for tests where we try to replicate how users would import the library):

```suggestion
import useBadge from '../useBadge';
```

Could you please update all the imports?